testing/routes/processors/main.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+  required_providers {
+    leanspace = {
+      source = "leanspace/leanspace"
+    }
+  }
+}
+
+variable "path" {
+  type        = string
+  description = "The path to the processor's source code file (a .jar)."
+}
+
+data "leanspace_processors" "all" {
+  filters {
+    ids = []
+    query = ""
+    page  = 0
+    size  = 10
+    sort  = ["name,asc"]
+  }
+}
+
+resource "leanspace_processors" "test_create_processor" {
+  file_path                            = var.path
+  version                              = "v1.0"
```

Comment:
Nit: no need for `v` it seems to be already displayed on the UI

Reply:
Done